card/db/migrate_core_cards/20191008201046_fixture_fix.rb

```diff
@@ -0,0 +1,9 @@
+# -*- encoding : utf-8 -*-
+
+class FixtureFix < Card::Migration::Core
+  def up
+    set_name = ["*header", :self]
+    set_card = ensure_card Card::Name[set_name] # set missing
+    Card[[set_name, :read]].update! left_id: set_card.id
```

Comment:
The absence of the `*header+*self` set was causing problems.  Without a set, the read rule wasn't getting in the cache.

I don't understand what changed to make this start showing up.  The migration that added the read rule was back in 2015 (`20150202143810_import_bootstrap_layout.rb`).  You didn't do any reseeding recently that would have re-run that migration, did you?